backends/server/package.json

```diff
@@ -5,11 +5,19 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "build": "yarn tsc",
+    "start": "node server.js",
```

Comment:
I think the "start" script is not needed no more/

Reply:
this is not the updated file, but "start" is for production, and "serve" is for development.